src/util/fs.js

```diff
@@ -58,15 +60,23 @@ const futimes: (fd: number, atime: number, mtime: number) => Promise<void> = pro
 const copyFile: (src: string, dest: string, flags: number, data: CopyFileAction) => Promise<void> = fs.copyFile
   ? // Don't use `promisify` to avoid passing  the last, argument `data`, to the native method
     (src, dest, flags, data) =>
-      new Promise((resolve, reject) => fs.copyFile(src, dest, flags, err => (err ? reject(err) : resolve(err))))
+      new Promise((resolve, reject) =>
+        fs.copyFile(src, dest, flags, err => {
+          if (err) {
+            reject(err);
+          } else {
+            fixTimes(0, dest, data).then(() => resolve(err)).catch(ex => reject(ex));
```

Comment:
Is the `err` in `resolve(err)` is a mistake?

Reply:
Maybe. I just preserved what the original code did:

```
new Promise((resolve, reject) => fs.copyFile(src, dest, flags, err => (err ? reject(err) : resolve(err))))
```

The code I was replacing would `resolve(err)` too. I didn't want to make the assumption that it wasn't intentional. :shrug:

Reply:
That's ... weird. I think we can just fix this and check that everything is still green.